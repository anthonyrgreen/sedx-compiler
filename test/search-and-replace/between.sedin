goodbye hey hey hey nananana