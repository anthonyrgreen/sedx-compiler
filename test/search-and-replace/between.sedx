let
  myAtMost := atMost('3', 'hey', maybe(' '))
  myAtLeast := atLeast('1', 'goodbye')
  myBetween := between('1', '4', 'na')
match
${capture(myAtLeast()) as myAtLeast} ${capture(myAtMost()) as myAtMost} ${capture(myBetween()) as myBetween}
substitute
${myBetween} ${myAtMost} ${myAtLeast}