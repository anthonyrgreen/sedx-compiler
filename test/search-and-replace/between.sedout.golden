nananana hey hey hey goodbye